index.html

```diff
@@ -9,9 +9,10 @@
       
       <!--Meta tags-->
       <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-      <meta name="keywords" content="Salman Shah, Intern, Data Science, Machine Learning">
+       <meta name="keywords" content="Real Estate , Apartments , Property , Houses for Sale , Ambernath , Affordable Homes ">
```

Comment:
Add these meta tags to the other pages as well, and check your mail for other meta tags that I've sent!